chapters/ch15.md

```diff
@@ -1,56 +1,96 @@
 
 Error analysis is an iterative process. Don’t worry if you start off with no categories in mind. After looking at a couple of images, you might come up with a few ideas for error categories. After manually categorizing some images, you might think of new categories and re-examine the images in light of the new categories, and so on.
 ->
 
+Phân tích lỗi là một quá trình lặp đi lặp lại. Đừng lo nếu bạn bắt đầu mà vẫn chưa nghĩ được hạng mục nào. Bạn sẽ có thêm ý tưởng mới về hạng mục lỗi sau khi phân tích một vài tấm ảnh. Sau khi phân loại thủ công một số hình ảnh, bạn có thể nghĩ ra các hạng mục mới và đối chiếu lại các mẫu ảnh theo hạng mục mới đó.
+
 
 Suppose you finish carrying out error analysis on 100 misclassified dev set examples and get the following:
 ->
 
+Giả sử bạn hoàn thành việc phân tích lỗi 100 mẫu bị phân loại nhầm trên tập phát triển và có được kết quả như sau:
 
 ![img](../imgs/C15_02.png)
+->
+
+|  Ảnh  | Chó | Họ Mèo | Mờ | Ghi chú|
+| :-----: | :-------: | :-------: |:-------: |:-------: |
+| 1       |  x  |     |     | Chó pitbull có màu lạ|
+| 2       |     |     |  x  |                      |
+| 3       |     |  x  |  x  | Ảnh sư tử chụp ở sở thú trong một ngày mưa |
+| 4       |     |  x  |     | Một con báo bị khuất sau cây |
+| ...     | ... | ... | ... | ... |
+| **Tổng %**  | 8% | 43% | 61% ||
 
 You now know that working on a project to address the Dog mistakes can eliminate 8% of the errors at most. Working on Great Cat or Blurry image errors could help eliminate more errors. Therefore, you might pick one of the two latter categories to focus on. If your team has enough people to pursue multiple directions in parallel, you can also ask some engineers to work on Great Cats and others to work on Blurry images.
 ->
 
+Bạn thấy rằng việc khắc phục lỗi phân loại nhầm trên hạng mục Chó có thể loại bỏ tối đa 8% lỗi. Khắc phục các lỗi trên hạng mục Họ Mèo và Mờ có thể loại bỏ được nhiều lỗi hơn. Vì vậy bạn có thể chọn một trong hai hạng mục trên để tập trung vào. Nếu nhóm của bạn có đủ nhân lực để khắc phục nhiều hạng mục lỗi song song, bạn có thể phân công một số kỹ sư khắc phục lỗi trên hạng mục Họ Mèo, những người còn lại khắc phục lỗi trên hạng mục Mờ.  
 
 Error analysis does not produce a rigid mathematical formula that tells you what the highest priority task should be. You also have to take into account how much progress you expect to make on different categories and the amount of work needed to tackle each one.
 ->
 
-
+Phân tích lỗi không tạo ra một công thức toán học cứng nhắc bắt buộc bạn phải tuân theo các thứ tự ưu tiên của nó. Bạn cũng cần đánh giá khả năng cải thiện mà bạn mong muốn trên các hạng mục cũng như khối lượng công việc cần thiết để giải quyết từng hạng mục đó.
```

Comment:
> bắt buộc bạn phải tuân theo các thứ tự ưu tiên của nó.

nên chăng đổi là "cho bạn biết hạng mục nào có độ ưu tiên cao nhất"? Theo như mình hiểu là "task" ở đây là "cải thiện error của từng hạng mục" (nói đến ở câu sau).

Reply:
1. "Error analysis does not produce a rigid mathematical formula that tells you what the highest priority task should be" = "Phân tích lỗi không tạo ra một công thức toán học cứng nhắc cho bạn biết nhóm lỗi nào nên được ưu tiên làm trước"? 

2. "expect" = "ước lượng/ ước chừng"??